src/demo/playground.js

```diff
@@ -1,47 +1,47 @@
-import React, { useEffect, useRef, useState } from 'react'
-import styled from 'styled-components'
-import { SegmentGroup, Segment, Button, BaseStyles, ColorDot } from './ui-system'
-import colors from 'nice-color-palettes'
-import { exampleNames } from './example-names'
-import Avatar from '../lib'
-import { CopyToClipboard } from 'react-copy-to-clipboard'
+import React, { useEffect, useRef, useState } from 'react';
+import styled from 'styled-components';
+import { SegmentGroup, Segment, Button, BaseStyles, ColorDot } from './ui-system';
+import colors from 'nice-color-palettes';
+import { exampleNames } from './example-names';
+import Avatar from '../lib';
+import { CopyToClipboard } from 'react-copy-to-clipboard';
```

Comment:
```suggestion
import React, { useEffect, useRef, useState } from 'react';
import styled from 'styled-components';
import { SegmentGroup, Segment, Button, BaseStyles, ColorDot } from './ui-system';
import colors from 'nice-color-palettes';
import { exampleNames } from './example-names';
import Avatar from '../lib';
import { CopyToClipboard } from 'react-copy-to-clipboard';
```

please ask the core team whether they want single quotes for they didn't have them from the start